plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -744,6 +744,28 @@ int RTMP_AddStream(RTMP *r, const char *playpath)
     return idx;
 }
 
+// Returns true if the string needs to be freed
+static char* get_hostname(AVal* host, bool* should_free)
```

Comment:
```suggestion
static char *get_hostname(AVal *host, bool *should_free)
```
\</clang-format\> (hopefully)

Reply:
this has been through several clang-format passes and it apparently did not catch that haha. i'll fix it.